scripts/setup-transformers.ts

```diff
@@ -117,8 +117,16 @@ writeFileSync('transformers.js/dist/transformers.js', transformersJs)
 
 console.log("Copying transformers.js to node_modules...")
 // re-construct phantom dependency in node_modules
-rmSync('node_modules/@xenova/transformers.js', { recursive: true });
-mkdirSync('node_modules/@xenova/transformers.js', { recursive: true });
+if (existsSync("./node_modules/@xenova/transformers.js")) {
+  rmSync("./node_modules/@xenova/transformers.js", { recursive: true });
+} else {
+  if (fs.readdirSync("node_modules").length === 0) {
```

Comment:
I believe the `fs` namespaced import is not available in local scope, as the functions are directly imported. It probably should be `readdirSync` instead.  Would lead to access of `Reading readdirSync of undefined` probably.